driver/src/main/com/mongodb/MongoClientOptions.java

```diff
@@ -290,6 +290,8 @@ public int getMaxWaitTime() {
      * The maximum idle time of a pooled connection.  A zero value indicates no limit to the idle time.  A pooled connection that has
      * exceeded its idle time will be closed and replaced when necessary by a new connection.
      *
+     * <p>Default is 0.</p>
```

Comment:
What does the default of 0 mean?

Change to `<p>Default is 0, indicating no limit to the idle time.</p>`